Scripts/SecretServer/Coveo/Discovery/readme.md

```diff
@@ -0,0 +1,138 @@
+# Coveo Account Discovery
+Add Disclaimer
+
+## Create Discovery Source
+.
+
+### Create SaaS Scan Template
+If this template has already been created in another Delinea Integration package please skip
+to the [Create Account Scan Template](#create-account-scan-template ) 
+
+- Log in to Secret Server Tenant
+- Navigate to **ADMIN** > **Discovery** > **Configuration** >   **Scanner Definition** > **Scan Templates** 
+- Click **Create Scan Template**
+- Fill out the required fields with the information
+    - **Name:** (Example: SasS Tenant)
+    - **Active:** (Checked)
+    - **Scan Type:** Host
+    - **Parent Scan Template:** Host Range
+    - **Fields**
+        - Change HostRange to **tenant-url**
+    - Click Save
+    - This completes the creation of the Saas Scan Template Creation
+ 
+
+### Create Account Scan Template
+
+- Log in to Secret Server Tenant
+- Navigate to **ADMIN** > **Discovery** > **Configuration** >   **Scanner Definition** > **Scan Templates** 
+- Click **Create Scan Template**
+- Fill out the required fields with the information
+    - **Name:** (Evxample: Coveo Account)
+    - **Active:** (Checked)
+    - **Scan Type:** Account
+    - **Parent Scan Template:** Account(Basic)
+    - **Fields**
+        - Change Resource to **tenant-url**
+        - Add field: Admin-Account (Leave Parent and Include in Match Blank)
+        - Add field: Service-Account (Leave Parent and Include in Match Blank)
+        - Add field: Local-Account (Leave Parent and Include in Match Blank)
+    - Click Save
+    - This completes the creation of the Account Scan Template Creation
```

Comment:
This should be moved left once space to align with "Fill out the required fields with the information"